src/drive/targets/manifest.webapp

```diff
@@ -175,5 +175,12 @@
       "verbs": ["POST"],
       "description": "Remote-doctype required to send anonymized measures to the DACC shared among mycozy.eu's Cozy."
     }
+  },
+  "accept_from_flagship": true,
+  "accept_documents_from_flagship": {
+    "accepted_mime_types": ["application/octet-stream"],
```

Comment:
this will not be the only accepted mime. In fact, with Drive, you can go with : `[*]`.

Reply:
Updated with "*/*" according to https://www.rfc-editor.org/rfc/rfc7231#section-5.3.2